### PR TITLE
Redesign

### LINKED ISSUE #16: Redesign Front facing
We redesigning the front facing website to give it a consistent look across pages.

XD Link: [Here](https://xd.adobe.com/view/3afdfa4b-ff11-40f6-5e98-c64345c2d8ce-4ccc/screen/fbb56416-c4d5-4027-8984-a4e049a7d7a4/About)

### PR DESCRIPTION
**Pull request by obasekietinosa**

### CODE REVIEW

**Comment by obasekietinosa** [comment]
Kindly do not merge. PR created for review only and to preview deployment.

**Review by vunderkind — COMMENTED** [review]
Good stuff, @obasekietinosa!

A few fixes: 
a. The footer is not fixed in place, and obscures various sections of the screen as you move about.
b. Consider adding a little margin to the sides of the other content as they're a little too close to the edges of the screen
c. In the /needhelp page, the button could use some work with styling. 
d. (This is a fix I'd implemented in master and your update will override them): the share button still says ten, instead of three people. 
e. Consider swapping out the h2 tags in the /needhelp page for <p> tags.
f. The 'be an angel' button does not respond to mouse hover. Makes it look like it's not 'wired'.
g. Consider reducing the padding on the buttons a tiny bit. 
h. The styling on the team page makes it a little difficult to know where one person's profile ends and the other begins. 

---
PS: love what you've done with standardizing the CSS!

File: src/components/SocialShare/SocialShare.js
Comment (oreHGA) [inline src/components/SocialShare/SocialShare.js:13]: typo?

**Review by oreHGA — COMMENTED** [review]
Nice work with the PR! I think I spotted a typo with the facebook fontawesome icon

File: src/components/SocialShare/SocialShare.js
Comment (obasekietinosa) [reply src/components/SocialShare/SocialShare.js:13]: No. The Facebook-f icon comes without the blue box around it. Made it fit better with the other icons.

**Comment by obasekietinosa** [comment]
@oreHGA @vunderkind Working on the other pages today. Should push soon.

**Comment by vunderkind** [comment]
> @oreHGA @vunderkind Working on the other pages today. Should push soon.

Excellent work! Well done, Etin.

**Comment by obasekietinosa** [comment]
@oreHGA @vunderkind requesting a final review before we merge?

**Comment by oreHGA** [comment]
Reviewing now

File: src/components/Header/Header.css
Comment (oreHGA) [inline src/components/Header/Header.css]:
```suggestion
.Linq {
    display: flex;
    justify-content: space-around;
    text-align: center;
    padding: 10px 0;
    height: 40px;
    box-shadow: var(--shadow);
}
```

Add the justification so that the header links space properly across mobile and desktop

**Review by oreHGA — CHANGES_REQUESTED** [review]
This is great man! Thanks a lot for taking this up.

Suggested only one change the header display and we should be good to go 🚀

**Review by oreHGA — APPROVED** [review]
Committed the header change we should be good to go now

**Pull request merged**
